### ISSUE TITLE
Sales order item not saved when creating project-task

### ISSUE THREAD

**Issue by PatrickMallee** [odoo/odoo#24592]
Impacted versions: V11 (latest Runbot version)

Steps to reproduce: Create task, select project, select sales order item. Save. 

Current behavior: you can see that the sales order item is not saved.

You have to edit the task again to make sure the sales order item is saved.

Expected behavior:sales order item is saved and linked to the project-task

Video/Screenshot link (optional):

**Comment by pedrobaeza** [comment]
I confirm that this happens.

@jem-odoo you write this piece of code: https://github.com/odoo/odoo/commit/8b3c3c0876805a812eda496a98b18e3c227f81f3#diff-674700f93bbb80c42896ddbb0cfaa760R64

You should check also if there's any value. I recommend you to change by this:
```python
        if values.get('parent_id'):
            values['sale_line_id'] = self.env['project.task'].browse(values['parent_id']).sudo().sale_line_id.id
```

**Comment by sswapnesh** [comment]
Duplicate of #22707

**Comment by pedrobaeza** [comment]
Closing then in favor of the other issue that has more comments about.

**Comment by pedrobaeza** [comment]
Solved in #22707
